lib/sidekiq_bus/adapter.rb

```diff
@@ -19,16 +25,40 @@ def redis(&block)
       end
 
       def enqueue(queue_name, klass, hash)
-        ::Sidekiq::Client.push('queue' => queue_name, 'class' => klass, 'args' => [hash])
+        ::Sidekiq::Client.push('queue' => queue_name,
+                               'class' => klass,
+                               'args' => [hash])
       end
 
       def enqueue_at(epoch_seconds, queue_name, klass, hash)
-        ::Sidekiq::Client.push('queue' => queue_name, 'class' => klass, 'args' => [hash], 'at' => epoch_seconds)
+        ::Sidekiq::Client.push('queue' => queue_name,
+                               'class' => klass,
+                               'args' => [hash],
+                               'at' => epoch_seconds)
       end
 
+      # Sets up the heartbeat to be broadcast via sidekiq. Only enable this when
+      # you have disabled the resque heart beat schedule as well, as having both
+      # may cause issues.
+      #
+      # While this will work so long as every time sidekiq boots it triggers this
+      # set up. You may consider enabling dynamic schedules to keep all nodes up
+      # to date if it ever changes.
       def setup_heartbeat!(queue_name)
-        # TODO: not sure how to do this or what is means to set this up in Sidekiq
-        raise NotImplementedError
+        require 'sidekiq-scheduler'
+
+        ::Sidekiq.set_schedule(
+          'sidekiqbus_heartbeat',
+          every: '1min',
```

Comment:
Does a string work here? Asking because in the docs it looks like [they are looking for an array](https://github.com/moove-it/sidekiq-scheduler#dynamic-schedule).

Reply:
It does actually work because they coerce the inputs:
https://github.com/moove-it/sidekiq-scheduler/blob/master/lib/sidekiq-scheduler/rufus_utils.rb